Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/__init__.py

```diff
@@ -24,7 +26,7 @@
         'exclude': ('darwin', # mac
                     'linux')  # linux, linux_x64
     }),
-    "doc_url": "https://github.com/o3de/o3de/blob/development/Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/readme.md"
+    "doc_url": DCCSI_DOCS_URL
```

Comment:
Should we publish this to the o3de.org documentation site?

Reply:
We don't have a landing page in the docs yet for this work.
But there are some instructions in the readme for the DccScriptingInterface.